src/design-tokens/mixins.css

```diff
@@ -133,13 +133,15 @@
    * Input error state
    */
   &.error {
+    /* TODO: should this color be applied when a field is in an :invalid state */
```

Comment:
Can we make input have an `invalid` state manually, or is it interpreted from fields such as the `disabled` attribute?

Reply:
`:invalid` is determined by the browser based on other attributes, and the value in the field, so it's not really an attribute unto itself. for `<textarea>`s `:invalid` applies if the value in the field is outside the bounds of what `minlength` and `maxlength` says is allowed (and in the case of `maxlength` only after some user action triggers the check, which is weird)

For `<input>`s, `:invalid` applies if you have a `required` field, and it becomes empty OR if the [`pattern`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#individual_attributes) is violated, along with the above. 

When i added `&:invalid` to the selector list on line 135, it changed some TextField snapshots to an error state b/c the fields didn't have any default value. this means that blank forms might start out with red outlines if marked as required, which we don't want